File: aoc2015/day1/part2.go
package main 

import "fmt"
import "log"
import "io/ioutil"

func main(){
	file,err := ioutil.ReadFile("input.txt")

	if(err!=nil){
		log.Fatal(err)
	}
	position := 0 

	for index,value := range string(file) {
		if  value == '('{
			position +=1
		} else if value == ')' {
			position -=1
		}

		if position == -1 {
			fmt.Println(index+1)
			break 
		}
	}
}

File: aoc2015/day1/part2.py
def main():
    with open('input.txt','r') as f:
        p=f.read()

    position = 0

    for key,value in enumerate(p):
        if value =='(':
            position+=1
        elif value ==')':
            position-=1

        if position == -1:
            print(key+1)
            break

main()
